docs/metrics/exemplars/README.md

```diff
@@ -1,99 +1,154 @@
 # Using Exemplars in OpenTelemetry .NET
 
+- [Install and run Jaeger](#install-and-run-jaeger)
+- [Install and run Prometheus](#install-and-run-prometheus)
+- [Install and configure Grafana](#install-and-configure-grafana)
+- [Export metrics and traces from the
+  application](#export-metrics-and-traces-from-the-application)
+- [Use exemplars to navigate from metrics to
+  traces](#use-exemplars-to-navigate-from-metrics-to-traces)
+- [Learn more](#learn-more)
+
 Exemplars are example data points for aggregated data. They provide specific
 context to otherwise general aggregations. One common use case is to gain
 ability to correlate metrics to traces (and logs). While OpenTelemetry .NET
 supports Exemplars, it is only useful if the telemetry backend also supports the
-capabilities. This tutorial uses well known open source backends to demonstrate
-the concept. The following are the components involved:
+capabilities. This tutorial uses well known open-source backends to demonstrate
+the concept. The following components are involved:
```

Comment:
nit: Maybe link the first "Exemplars" reference here to `customizing-the-sdk/README.md#exemplars`

Reply:
Added, thanks!